packages/core/src/App/Components/Layout/Header/toggle-menu-drawer.jsx

```diff
@@ -313,7 +342,204 @@ const ToggleMenuDrawer = React.forwardRef(
+                                                    className={classNames({
+                                                        'header__menu--back-to-ui': !is_dark_mode,
+                                                        'header__menu--back-to-ui--dark': is_dark_mode,
+                                                    })}
+                                                    type='button'
+                                                    large
+                                                    onClick={() => {
+                                                        setIsPreAppStore(false);
+                                                        toggleDrawer();
+                                                    }}
+                                                >
+                                                    <Text
+                                                        className={classNames({
+                                                            'header__menu--back-to-ui-text': !is_dark_mode,
+                                                            'header__menu--back-to-ui-text--dark': is_dark_mode,
+                                                        })}
+                                                        size='xs'
+                                                    >
+                                                        {localize('Back to old interface')}
+                                                    </Text>
+                                                    <Icon
+                                                        className='header__menu-mobile-rightarrow'
+                                                        icon='IcArrowRight'
+                                                        size={18}
+                                                        color='red'
+                                                    />
+                                                </Button>
+                                            </MobileDrawer.Item>
+                                        )}
+                                        {is_logged_in && (
+                                            <MobileDrawer.Item>
+                                                <MenuLink
+                                                    link_to={routes.trading_hub}
+                                                    icon='IcAppstoreHome'
+                                                    text={localize('Trading Hub')}
+                                                    onClickLink={toggleDrawer}
+                                                    changeCurrentLanguage={changeCurrentLanguage}
+                                                />
+                                            </MobileDrawer.Item>
+                                        )}
+                                        {is_logged_in && location === routes.trade && location === routes.trade && (
```

Comment:
Trade should be visible for all pages rather than `Trading hub`, `cashier` and `account setting`

Reply:
same bahaviour for platform switcher

Reply:
Will do

Reply:
Done